lib/src/main/kotlin/cz/eman/swagger/codegen/generator/kotlin/KotlinClientCodegen.kt

```diff
@@ -470,6 +523,28 @@ open class KotlinClientCodegen : org.openapitools.codegen.languages.KotlinClient
         }
     }
 
+    /**
+     * Processes options for API generated classes. Removes all API template files from the generation isf this
+     * option is set to false (do not generate API). For more information see [initSettingsInfrastructure].
+     *
+     * @since 2.2.4
+     */
+    private fun processOptsApi() {
+        var generateApi = true
+        if (additionalProperties.containsKey(GENERATE_API)) {
+            generateApi = convertPropertyToBooleanAndWriteBack(GENERATE_API)
+        }
+
+        if (!generateApi) {
+            apiDocTemplateFiles.clear()
+            apiTemplateFiles.clear()
+            if (additionalProperties.containsKey(GENERATE_API)) {
```

Comment:
Asi nechápu ale není tahle kontrola zbytečná? To máš obražené v tom generateApi = řádku.